package.json

```
@@ -58,7 +60,7 @@
     "@octokit/webhooks": "13.2.8",
     "@octokit/webhooks-types": "7.5.1",
     "@sinclair/typebox": "0.32.35",
-    "@ubiquity-dao/ubiquibot-logger": "1.3.0",
+    "@ubiquity-dao/ubiquibot-logger": "1.3.1",
```

```suggestion
    "@ubiquity-dao/ubiquibot-logger": "*",
```

We should do this until things are stable.

---

I think this is a really bad idea considering that major version imply breaking changes and would break the whole project. What would be better would be `"1.x"` so only minor and patches are taken into account.

---

The point is to catch problems as soon as they occur. Testing in production is the fastest way to tighten our feedback loop. Realistically we would notice the problem and triage likely within 12 hours max. Realistically, pretty much immediately.

---

Yes but in this case that would break all the plugins that worked on a given version, which seems pretty heavy collateral damage for a bug because it would mean all plugins are broken until a fix is provided.

---

Can always revert in case of emergency. It takes a few secones.

---

We cannot revert, we should delete the latest deployed version on `npm` instead.

---

Would be pretty funny to automatically delete the latest on fatal crash. Almost like genetic algorithms.

---

This would imply that all plugins have access to `npm` publishing, and all would have to rebuild themselves. That is also why I am against using wildcards on packages, because maybe one version works fine for a plugin but the other crashes, which means on an revert maybe both plugins then crash. Also during the time we delete the latest version, all the plugins importing this package might break. When we pin packages only the erroneous plugin would break.

---

Wouldn't the kernel fail to build if there are any major changes or if the tests fail? Anyway I think it's fine if we do `^1.3.1`

---

Very likely. Still means that we would have to fix it before any other update is done.